pkgs/applications/misc/cobang/default.nix

```diff
@@ -99,6 +99,6 @@ buildPythonApplication rec {
     license = licenses.gpl3Only;
     maintainers = [ ];
```

Comment:
Would you mind adding yourself to maintainers? Or you can add me to maintainers

Reply:
I will try to add myself (and you) as maintainers, I just need to get myself added to `maintainers/maintainer-list.nix`.
I have made a https://github.com/NixOS/nixpkgs/pull/331326 which I am waiting on

Reply:
Cherry-pick that commit here. And respect order of commits.

Reply:
@superherointj I am trying my best to respect the project, I read the document about how to add oneself as a maintainer, but there is a lot left out, like what the subject of the pull request should be. Maybe it was written somewhere else, but when I did not find it.

Reply:
> @superherointj I am trying my best to respect the project, I read the document about how to add oneself as a maintainer, but there is a lot left out, like what the subject of the pull request should be. Maybe it was written somewhere else, but when I did not find it.

**Familiarity**. You cannot guess right what you don't know. But yes, documentation could be better, but realize, different people are at different level at different matters, so there isn't a documentation that fits everyone needs. Some know git or GitHub well, others don't... Most conventions are straightforward. If you want to change something, open a PR, reviewers will debate it [but is a tiring process].

In truth, everything sucks, proposing code sucks, reviewing sucks, committing sucks even more. (Having to own shit sucks.) This is unpaid work, right?

I ask myself everyday why I'm still doing this. Idk. I honestly don't know. =]

If a problem arise, we'll figure out a solution. Don't worry.

Reply:
@superherointj thanks for the link (I will read it in more details and may have follow-up questions), I think that there probably should be a link to the conventions in the document
https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
Because it was there I reading about how to add myself as an maintainer.

Reply:
I agree that documentation could be improved.

Reply:
@Aleksanaa 
Okay, I should have fixed the order of the commits (and the commit text there I added myself to the maintainer list)

@superherointj 
I hope everything looks good now